### PR TITLE
Fix Spellings and Typos

### PR DESCRIPTION
**Pull request by abhidixi11**
*Issue #, if available:*

*Description of changes:*
For Standard and Platform platform directories. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: libraries/standard/common/src/iot_taskpool.c [hunk 1]
```diff
@@ -1435,7 +1435,7 @@ static void _signalShutdown( _taskPool_t * const pTaskPool,
     /* Set the exit condition. */
     pTaskPool->maxThreads = 0;
 
-    /* Broadcast to all active threads to wake-up. Active threads do check the exit condition right after wakein up. */
+    /* Broadcast to all active threads to wake-up. Active threads do check the exit condition right after wakeing up. */
```
Comment (leegeth) [inline libraries/standard/common/src/iot_taskpool.c]: waking

File: libraries/standard/mqtt/include/iot_mqtt_serialize.h [hunk 2]
```diff
@@ -426,7 +426,7 @@ IotMqttError_t IotMqtt_GetPublishPacketSize( IotMqttPublishInfo_t * pPublishInfo
  *    xResult = IotMqtt_GetPublishPacketSize( &xMQTTPublishInfo, &xRemainingLength, &xPacketSize );
  *    IotMqtt_Assert( xResult == IOT_MQTT_SUCCESS );
  *    // Make sure the packet size is less than static buffer size
- *	  configASSERT( xPacketSize < mqttexampleSHARED_BUFFER_SIZE );
+ *	  IotMqtt_Assert( xPacketSize < mqttexampleSHARED_BUFFER_SIZE );
```
Comment (leegeth) [inline libraries/standard/mqtt/include/iot_mqtt_serialize.h]: why is this changed?

Comment (abhidixi11) [reply libraries/standard/mqtt/include/iot_mqtt_serialize.h]: Because it is CSDK and not FreeRTOS.
